src/content/blog/2025_ucc_unitaryhack.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Community-Driven Quantum Compilation w/ UCC @ UnitaryHACK 2025 
+author: Jordan Sullivan, Gregory Varghese, Amir Ali Malekani Nezhad, Marc Davis
+day: 31
+month: 7
+year: 2025
+tags: 
+  - python
+  - compiler
+  - qiskit
+  - qbraid
+  - bqskit
+  - quantum software
+  - hackathon
+  - open source
+---
+This year marks our fifth annual UnitaryHACK hackathon (check out our [post-event wrap-up blog](https://unitary.foundation/posts/2025_uhack/))! It's also the very _first_ year Unitary Foundation's new open-source quantum compiler [UCC](https://github.com/unitaryfoundation/ucc) has been a part of the event. In this blog, we're going to let three of the **UCC UnitaryHACK Bounty winners** tell you about their projects.
+
+<div style="display: flex; justify-content: center; align-items: center; margin: 20px 0;">
+  <img src="/images/2025_ucc_unitaryhack/uhack-ucc-2025-logo.png" alt="UnitaryHACK 2025 UCC logo" style="max-width: 700; width: 100%; height: auto;"/>
+</div>
+
+We created UCC to be a quantum compiler that's genuinely community-driven -- that's part of our mission to make quantum computing more accessible and beneficial to the most people -- and so we did something a little different from the typical [UnitaryHACK bounties](https://unitaryhack.dev/bounties/). We put out an [open call](https://github.com/unitaryfoundation/ucc/issues/381) for new quantum compiler passes, and the community delivered! Let's hear from them:
+
+#### Verifying logical equivalence  of quantum circuits
+[GH issue](https://github.com/unitaryfoundation/ucc/issues/62)  
+Bounty: $200  
+Closed by: [WingCode](https://github.com/WingCode)  
+
+**My name is Gregory Varghese**, I have a Undergraduate degree in Computer Science & Engineering, and I work as a Senior Backend Expert, currently focused on QML methods for chemical simulations. 
+
+**I wanted to work on this issue because** I am fascinated by this as fundamental problem at the intersection of quantum computing and computational complexity.
```

Comment:
I think a little blurb about what the issue was about would be helpful. I wouldn't expect many readers to go read each issue description.